### PR TITLE
interviews: publish our Dev Advocate interview guide

### PR DESCRIPTION
**Pull request by nicks**
Hello @ellenkorbes, @dbentley,

Please review the following commits I made in branch nicks/devrel:

e8d6ceff8dce8205fe288d8de4e1a96f4a8e0929 (2021-06-30 12:22:56 -0400)
interviews: publish our Dev Advocate interview guide

Code review reminders, by giving a LGTM you attest that:

* Commits are adequately tested
* Code is easy to understand and conforms to style guides
* Incomplete code is marked with TODOs
* Code is suitably instrumented with logging and metrics

### CODE REVIEW

**Comment by nicks** [comment]
FYI - this is the interview guide we used for L (and for other devrel candidates we were talking to at that time)

Since then, we've started the practice of publishing our interview guides in advance, then pointing candidates at them so they know what to expect. People have really appreciated it. So let's keep doing that!

**Comment by ellenkorbes** [comment]
LGTM content-wise but after you merge I'll open a copyedit PR 🤓

**Review by ellenkorbes — APPROVED** [review]

**Comment by nicks** [comment]
:grimacing:

**Pull request merged**
